README.md

```diff
@@ -183,9 +183,7 @@ $ ./yaml2json.sh sample.yaml > sample.json
 1. Install Rust from https://www.rust-lang.org/ if you don't have it already.
 2. Clone this repo locally.
 3. Grab your OpenAI API key and set the `OPENAI_API_KEY` environment variable with it.
-4. Run `cargo build --release` to compile the release binary to `target/release/pk`.
-5. Move the binary to wherever you like and add it to your `PATH`.
-5. Run the `pk` binary using the use cases and examples above for guidance.
+4. Run `cargo install --path .`
```

Comment:
This lets cargo manage PATH.

Reply:
This will install it just in this directory right? Like it won't add it to your PATH?

Reply:
Yeah I dunno what happened when I tried this a few nights ago. I just tried it again and it worked. Very cool!